fighters/common/src/opff/floats.rs

```diff
@@ -241,6 +242,10 @@ pub unsafe fn float_effects(fighter: &mut L2CFighterCommon, boma: &mut BattleObj
                 else if timer % 15 == 0 {  // every 15 frames
                     // drain 1 bar of Elwind
                     WorkModule::dec_int(boma, *FIGHTER_REFLET_INSTANCE_WORK_ID_INT_SPECIAL_HI_CURRENT_POINT);
+                    if timer % 30 == 0 {
```

Comment:
This should be in an `else if` block outside of this `else if` block.

Reply:
wouldn't that make it not run bc if it's divisible by 30 w/ no remainder it should always be divisible by 15 w/ no remainder right? or does code not work like that

Reply:
my b, not `else if`, just `if`. I'd rather the logic be contained in its own block

but ya, ur right